### PR TITLE
🐛Stops incorrect titles appearing with mouse hover on story bookend

### LINKED ISSUE #17654: AMP Story bookend mouseover title
In bookend, when you mouseover a link or thumbnail image, a title pups up. It's same for all the links.

### PR DESCRIPTION
**Pull request by addieachan**
- Fixes #17654

### CODE REVIEW

**Comment by addieachan** [comment]
@gmajoulet @newmuis Sorry if I misinterpreted anything but this is my version of the fix so PTAL :)

File: extensions/amp-story/1.0/amp-story.js [hunk 1]
```diff
@@ -279,6 +279,9 @@ export class AmpStory extends AMP.BaseElement {
     /** @private {boolean} */
     this.pausedStateToRestore_ = false;
 
+    /** @private {string} */
```
Comment (addieachan) [inline extensions/amp-story/1.0/amp-story.js]: Should this be private?

File: extensions/amp-story/1.0/amp-story.js [hunk 2]
```diff
@@ -811,6 +815,17 @@ export class AmpStory extends AMP.BaseElement {
...
 
+  /**
+   * Stores the title attribute value in memory then sets it to an empty string
+   * in order to avoid incorrect titles during mouse hover in the bookend.
+   * @private
+   */
+  extractTitle_() {
+    if (this.element.hasAttribute('title')) {
+      this.title_ = this.element.getAttribute('title');
+      this.element.setAttribute('title', '');
```
Comment (newmuis) [inline extensions/amp-story/1.0/amp-story.js]: I think we can just use `removeAttribute`.

**Comment by addieachan** [comment]
Ok sounds good! Also, I was talking to @gmajoulet, and we were discussing that storing the title in the .title_ field was unnecessary since I couldn't think of/find any places in the codebase that use the story's title. Do you think I should remove storing the title at all, and just remove the attribute?

File: extensions/amp-story/1.0/amp-story.js [hunk 3]
```diff
@@ -347,6 +350,7 @@ export class AmpStory extends AMP.BaseElement {
       this.analytics_.onNavigationStateChange(stateChangeEvent);
     });
 
+    this.extractTitle_();
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story.js]: If we don't need to store the title, could we just have a `this.element.removeAttribute('title')` here?
Maybe it's worth adding a super small comment and link the github issue?

Comment (newmuis) [reply extensions/amp-story/1.0/amp-story.js]: I would also be okay with this, then the follow-up PR to actually propagate the 1.0 metadata though to the bookend etc can worry about storing it.

File: extensions/amp-story/1.0/amp-story.js [hunk 4]
```diff
@@ -347,6 +347,10 @@ export class AmpStory extends AMP.BaseElement {
       this.analytics_.onNavigationStateChange(stateChangeEvent);
     });
 
+    // Removes title in order to prevent incorrect titles appearing on link
+    // hover. (See www./github.com/ampproject/amphtml/issues/17654)
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story.js]: Nit: You can just do (See #17654)

**Comment by addieachan** [comment]
Ping for merge @gmajoulet @newmuis :)

**Pull request merged**